pytest_dockerctl/__init__.py

```diff
@@ -13,8 +17,26 @@ class DockerClientError(Exception):
     """Generic docker API error.
     """
 
+def has_attr(
+    cntr: Container,
+    attr_path: Tuple[str]
```

Comment:
much better 😎 .

I've started trying to do this everywhere since it saves a reader from having to guess which object is being used, and further if you have ctags (or something similar setup) you can jump to the def.